sci-mathematics/scilab/scilab-5.5.2-r1.ebuild

```
@@ -0,0 +1,298 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+JAVA_PKG_OPT_USE="gui"
+VIRTUALX_REQUIRED="manual"
+
+inherit autotools bash-completion-r1 check-reqs eutils flag-o-matic \
```

```suggestion
inherit autotools check-reqs eutils flag-o-matic \
```

There's not really any point in having this USE flag, I already removed it from the other ebuilds, because we shouldn't have USE flags toggle the installation of small files. There is no point in forcing an user to recompile the entire package just to add bash coimpletions

---

not that line, but on the IUSE :-)

---

ooh oops, my bad. It should indeed by IUSE of course, we do still need the eclass